iis/iis.go

```diff
@@ -145,6 +175,44 @@ func getVersion() (string, error) {
 	}
 }
 
+// Gets a version object of InetMgr.exe which parses major.minor.build.revision string
+func getVersion() (*iisVersion, error) {
+	if versionStr, err := getVersionStr(); err != nil {
+		return nil, fmt.Errorf("Failed to get version string for iisVersion parsing: %v", err)
+	} else {
```

Comment:
nitpick: No need for `else` here. Just a suggestion, feel free to ignore it.

Reply:
I use the else here to keep within the scope of `versionStr`, but I can change it to just initializing the vars outside the if/else scope.